src/Sql/Sql/help/Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy.md

```diff
@@ -0,0 +1,250 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/set-azsqlinstancedatabasebackupshorttermretentionpolicy
+schema: 2.0.0
+---
+
+# Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy
+
+## SYNOPSIS
+Sets a backup short term retention policy.
+
+## SYNTAX
+
+### PolicyByResourceInstanceDatabaseSet (Default)
+```
+Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy [-ResourceGroupName] <String> [-InstanceName] <String>
+ [-DatabaseName] <String> [-DeletionDate <DateTime>] [-RetentionDays] <Int32>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### PolicyByInputObjectSet
+```
+Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy [-InputObject] <AzureSqlManagedDatabaseBaseModel>
+ [-RetentionDays] <Int32> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### PolicyByResourceIdSet
+```
+Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy [-ResourceId] <String> [-RetentionDays] <Int32>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy** cmdlet gets the short term retention policy registered to this database.
+The policy is the retention period, in days, for point-in-time restore backups.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy -ResourceGroupName resourcegroup01 -InstanceName server01 -DatabaseName database01 -RetentionDays 35
+ResourceGroupName : resourcegroup01
+InstanceName      : instance01
+DatabaseName      : database01
+DeletionDate      :
+RetentionDays     : 35
+```
+
+This command sets the short term retention policy for database01 to 35 days.
+
+### Example 2
+```powershell
+PS C:\> Get-AzSqlDatabase -ResourceGroupName resourcegroup01 -InstanceName server01 -DatabaseName database01 | Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy -RetentionDays 35
+ResourceGroupName : resourcegroup01
+InstanceName      : instance01
+DatabaseName      : database01
+DeletionDate      :
+RetentionDays     : 35
+```
+
+This command sets the short term retention policy for database01 to 35 days via piping in a database object.
+
+### Example 3
+```powershell
+PS C:\> Set-AzSqlDeletedInstanceDatabaseBackup -ResourceGroupName "ContosoResourceGroup" -InstanceName "ContosoServer" -DatabaseName "DB1" | Get-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy -RetentionDays 8
```

Comment:
I think you meant: Get-AzSqlDeletedInstanceDatabaseBackup -ResourceGroupName "ContosoResourceGroup" -InstanceName "ContosoServer" -DatabaseName "DB1" | Set-AzSqlInstanceDatabaseBackupShortTermRetentionPolicy -RetentionDays 8